lib/defaultMetrics.js

```diff
@@ -31,57 +33,54 @@ const metrics = {
 };
 const metricsList = Object.keys(metrics);
 
-let existingInterval = null;
-// This is used to ensure the program throws on duplicate metrics during first run
-// We might want to consider not supporting running the default metrics function more than once
-let init = true;
-
 module.exports = function startDefaultMetrics(config) {
```

Comment:
```suggestion
module.exports = function collectDefaultMetrics(config) {
```

maybe? The name makes no real difference